### PR TITLE
Fix event edit datetime format bug and improve event edit form UI

### PR DESCRIPTION
**Pull request by alexander-zw**
Added datetime format before data population happens, changed form template a bit, extracted out a constant string into utils

### CODE REVIEW

File: hknweb/events/views.py [hunk 1]
```diff
@@ -151,6 +152,14 @@ class EventUpdateView(UpdateView):
     form_class = EventUpdateForm
     template_name_suffix = '_edit'
 
+    def get_initial(self):
```
Comment (Rishabhk2357) [inline hknweb/events/views.py:155]: Where is this function used?

File: hknweb/settings/common.py [hunk 2]
```diff
@@ -11,8 +11,10 @@
 """
 import os
 from django.conf.global_settings import DATETIME_INPUT_FORMATS
+from hknweb.utils import DATETIME_12_HOUR_FORMAT
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+
```
Comment (Rishabhk2357) [inline hknweb/settings/common.py:17]: Unnecessary line add (not crucial to change, but would be nice)

File: hknweb/events/templates/events/event_edit.html [hunk 3]
```diff
@@ -40,7 +40,19 @@ <h1 id="add-event-title"> Edit Event: {{ event.name }} </h1>
 <div class="centered">
     <form method="post">
         {% csrf_token %}
-        {{ form.as_p }}
+        {% for field in form %}
```
Comment (Rishabhk2357) [inline hknweb/events/templates/events/event_edit.html:43]: What was wrong with the formatting produce by "form.as_p"

File: hknweb/events/views.py (hunk 1 above)
Comment (chevin-ken) [reply hknweb/events/views.py:155]: I believe get_initial is something that Django uses to populate the form with pre-existing data before it is actually rendered on the template

Comment (alexander-zw) [reply hknweb/events/views.py:155]: Yeah so it's used in the parent class

File: hknweb/settings/common.py (hunk 2 above)
Comment (alexander-zw) [reply hknweb/settings/common.py:17]: Huh not sure where that came from

**Pull request merged**